package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-promised",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
Change this back please. After I merge your pull request I will run `npm version minor` ([docs](https://docs.npmjs.com/cli/version)) to increment the version number for me before I `npm publish`.

This is the preferred way to go because for instance I may want to add another feature to the project this afternoon and then I could ship them together as one version change.